piyushgarg-dev/piyushgargdev-nextjs
# Fixes #309 Add Playlists

## What does this PR do?


I add all the playlists which are available at Piyush Garg YouTube channel until the date 24-08-2023. This Feature is very helpful for other to see courses which are free and available at YouTube and it look cool too.

Fixes #309 



https://github.com/piyushgarg-dev/piyushgargdev-nextjs/assets/111944042/89e9aefb-b8f6-4001-ad35-8c5ca7491ac3


## Type of change



- New feature (non-breaking change which adds functionality)

## How should this be tested?



1. Go on home page
2. Scroll down and see playlists and you can click and redirect to youtube playlist

## Mandatory Tasks


## Checklist
